worker/src/lib/dbLogger/DBLoggable.ts

```diff
@@ -642,21 +642,42 @@ export class DBLoggable {
     }
 
     const responseResult = await this.readAndLogResponse(db.queue);
+    const model =
```

Comment:
I think the business logic around images/s3 should be abstracted elsewhere. 

Could we make a new Manager class? That class can orchestrate this business logic?

It could also contain a lot of the business logic that is inside the S3Client.

Reply:
Something like this:

```typescript
export type RequestResponseContent = {
  requestId: string;
  organizationId: string;
  requestBody: string;
  responseBody: string;
};

export class S3Manager {
  constructor(private s3Client: S3Client) {}

  // Handle storing request/response data in S3
  // Handle storing images
  async storeRequestResponseData(
    content: RequestResponseContent
  ): Promise<Result<string, string>> {}

  async storeImage(): Promise<Result<string, string>> {
    
  }
}

```

Reply:
Ideally DbLoggable can just do this:

```typescript
 if (S3_ENABLED === "true") {
      const s3Manager = new S3Manager(db.s3Client);
      await s3Manager.storeRequestResponseData(params);
```